docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -420,7 +420,7 @@ docker run --rm \
 	-v /path/to/data:/data:rw \
 	--name flow-go \
 	--network host \
-	gcr.io/flow-container-registry/execution:v0.21.3 \
+	gcr.io/flow-container-registry/execution:v0.22.8 \
```

Comment:
This version will need to change once this is merged https://github.com/onflow/flow-go/pull/1440

Reply:
ok I will hold off on merging this.